src/features/public/PublicUserProfile/components/ProjectSnippet.tsx

```diff
@@ -1,90 +0,0 @@
-import React, { ReactElement } from 'react';
-import Sugar from 'sugar';
-import getImageUrl from '../../../../utils/getImageURL';
-import styles from '../../Donations/styles/Projects.module.scss';
-import i18next from '../../../../../i18n';
-import getFormatedCurrency from '../../../../utils/countryCurrency/getFormattedCurrency';
-
-const { useTranslation } = i18next;
-interface Props {
-  project: any;
-  key: number;
-}
-
-export default function ProjectSnippet({ project, key }: Props): ReactElement {
```

Comment:
I just put quite some effort in makes the PR https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/287 work which is doing some changes in this file - so where is this code going to? How is the TPO page done now?

Reply:
We are using the same component used for showing the home page of the app, thanks to the new API format